packages/demo/pages/editable-html.js

```diff
@@ -101,7 +102,9 @@ class RteDemo extends React.Component {
       disabled: false,
       width: '',
       height: '',
-      keypadMode: 'everything'
+      keypadMode: 'everything',
+      markupText: html,
+      htmlHasText: true
```

Comment:
this name isn't clear to me - why not just `hasText`? The client comp doesn't care if it's html or not?

Reply:
i'll change it